lib/nerves_motd.ex

```diff
@@ -14,10 +14,20 @@ defmodule NervesMOTD do
   \e[34m███▌    \e[36m▀▜████\e[0m
   """
 
+  @typedoc """
+  MOTD options
+  """
+  @type option() :: {:logo, String.t() | false}
+
   @doc """
   Print the message of the day
+
+  Options:
+
+  * `:logo` - a custom logo to display instead of the default Nerves logo. Pass
+    `false` or `""` for no logo.
```

Comment:
I think `""` for no logo can be different from `false`, because `""` is truthy. I wonder if we need `""` option for no logo 🤔

Reply:
Yeah.... I was almost ready to remove `false` when looking at the specs, since passing `true` raises. The empty string renders the same thing as `false`, and it seemed simpler to just say that the argument is the logo. The only place in Elixir that lets you pass `false` or a string that I can think of is `@doc` and `@moduledoc`. The idiom of passing `false` to skip rendering a string rather than just passing `""` is not something that I'm that familiar with, but I'm ok if this is a thing.

Reply:
I'm going to update this is in a PR shortly based on our Slack conversation...